docs/core/native-interop/exposing-dotnet-components-to-com.md

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
File name: for SEO and general guidelines (https://review.docs.microsoft.com/en-us/help/contribute/file-path-guidelines?branch=master#name-your-files)
I think the file name of this article should be expose-components-to-com.md (dotnet is already part of the path and shouldn't be repeated if possible)